docker/statutory/Dockerfile

```diff
@@ -20,9 +16,11 @@ USER node
 ENV NPM_CONFIG_PREFIX=/home/node/.npm-global
 ENV PATH="/home/node/.npm-global/bin:${PATH}"
 
-RUN npm install -g --loglevel warn nodemon bunyan && npm cache clean --force
-RUN npm install --loglevel warn
+RUN npm install -g nodemon@2.0.6
+RUN npm install -g bunyan@1.8.14
+RUN npm cache clean --force
+RUN npm install
 
-CMD sh /usr/app/scripts/bootstrap.sh && nodemon -e "js,json" lib/run.js
+CMD ["sh", "/usr/app/scripts/bootstrap.sh && nodemon -e 'js,json' lib/run.js"]
```

Comment:
not sure if this'll work as there are 2 different commands

Reply:
Yeah, I was afraid for that already. CMD should only do 1 command though. Maybe @linuxbandit knows more on this

Reply:
best would be to have ENTRYPOINT *and* CMD. I have an example on a refactoring of core dockerfile, I'll show you

Reply:
https://github.com/AEGEE/core/pull/195

Reply:
Splitting them is strange, so I've put a FIXME there so we can look into it later. See for more info; https://github.com/AEGEE/core/pull/195#issuecomment-775117919